### PR TITLE
Fix error messages for easy understanding

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What?
- Fix error messages for easy understanding

## Why?
- エラーメッセージが分かりやすい方が使いやすいので

## See also
- Related https://github.com/dataware-tools/dataware-tools/issues/22

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/api-meta-store/pull/53#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapi-meta-store).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/api-meta-store/blob/main/OWNERS)~~ [WatanabeToshimitsu]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: api/config.py [hunk 1]
```diff
@@ -129,9 +129,10 @@ def _update_config(database_id: str, config):
 
         # Check contents in each columns
         for column in config['columns']:
-            assert 'name' in column.keys()
-            assert 'dtype' in column.keys()
-            assert 'aggregation' in column.keys()
+            assert 'name' in column.keys(), 'config for columns should have "name" key'
```
Comment (d-hayashi) [inline api/config.py:132]: `should` -> `must` ですね

File: api/utils.py [hunk 2]
```diff
@@ -425,5 +425,12 @@ def get_check_permission_client(authorization: str = Header(None)):
         return DummyCheckPermissionClient(authorization)
     return CheckPermissionClient(authorization)
 
+
+def check_db_output_schema(dict: dict) -> None:
+    """Check specified value is list
+    """
+    assert ('data' in dict.keys() and isinstance(dict['data'], list)), \
+        'database returned data as unexpected schema'
```
Comment (d-hayashi) [inline api/utils.py:433]: データベースに問題があるというよりはPyDTK側での処理に問題がある場合なので、単純に `unexpected data was returned by PyDTK` とかで良いかも？

**Review by d-hayashi — COMMENTED** [review]
修正あざます！
コメント部分以外はLGTMです！

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/api-meta-store/pull/53#issuecomment-891672755" title="LGTM">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/api-meta-store/pull/53#" title="Author self-approved">WatanabeToshimitsu</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapi-meta-store).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/api-meta-store/blob/main/OWNERS)~~ [WatanabeToshimitsu,d-hayashi]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**